src/XGBoostSharp.Tests/XGBClassifierTest.cs

```diff
@@ -69,6 +70,25 @@ public void XGBClassifierTest_SaveAndLoad()
         TestUtils.AssertAreEqual(expected, actual);
     }
 
+    [TestMethod]
+    public void XGBClassifierTest_SaveAndLoadRaw()
```

Comment:
@madskonradsen Seems the test fails on CI:
https://github.com/mdabros/XGBoostSharp/actions/runs/11380712392/job/31695853717?pr=32
```
 Failed XGBClassifierTest_SaveAndLoadRaw [29 ms]
  Error Message:
   Assert.AreEqual failed. Expected a difference no greater than <0.0001> between expected value <0.2789577> and actual value <0>. 
  Stack Trace:
     at XGBoostSharp.Test.TestUtils.AssertAreEqual(Single[] expected, Single[] actual) in /home/runner/work/XGBoostSharp/XGBoostSharp/src/XGBoostSharp.Tests/TestUtils.cs:line 26
   at XGBoostSharp.Test.XGBRegressorTest.XGBClassifierTest_SaveAndLoadRaw() in /home/runner/work/XGBoostSharp/XGBoostSharp/src/XGBoostSharp.Tests/XGBRegressorTest.cs:line 72
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
   at System.Reflection.MethodBaseInvoker.InvokeWithNoArgs(Object obj, BindingFlags invokeAttr)
```

Can you take a look at this before I start reviewing?

Reply:
Sure, that was a just a simple mixup :) Nice that CI has been enabled.